sdk/src/test/java/io/dapr/it/DaprIntegrationTestingRunner.java

```diff
@@ -85,6 +85,10 @@ public DaprFreePorts initializeDapr() throws Exception {
   private String buildDaprCommand(){
     StringBuilder stringBuilder= new StringBuilder(String.format(DAPR_RUN, this.appName))
       .append(this.useAppPort ? "--app-port " + this.DAPR_FREEPORTS.appPort : "")
+            .append(" --grpc-port ")
+            .append(this.DAPR_FREEPORTS.grpcPort)
+            .append(" --port ")
+            .append(this.DAPR_FREEPORTS.httpPort)
```

Comment:
Do we need this ? dapr cli finds the free ports for http and grpc and set the envrionment variables by itself. 

@LMWF

Reply:
yes, but IT does not aware of that, does not read the environment variables

Reply:
JuanJose is correct. We need to know the ports outside Dapr too.

Reply:
If the IT needs to know what they are then this is fine.  Otherwise it would be preferable if dapr decides.

Note having hyphens in the command line here is only a problem if they're inside the -Dexec.args portion of the mvn command.  Here they are not.  If they were being used to populate DAPR_COMMAND (the immediate next line), they would and that would be a problem.